Source/Core/BoxGeometry.js

```diff
@@ -269,6 +269,10 @@ define([
         var indices;
         var positions;
 
+        if ((max.x <= 0) || (max.y <= 0) || (max.z <= 0)) {
```

Comment:
This isn't exactly right.  We define a box using the positions of a corner on the top of the box, and the opposite corner on the bottom of the box.  So the box would be empty when `min === max`